lib/appoptics_apm/oboe_init_options.rb

```diff
@@ -151,7 +153,7 @@ def read_and_validate_service_key
     end
 
     def validate_token(token)
-      if (token !~ /^[0-9a-fA-F]{64}|[0-9a-zA-Z_\-]{71}$/)
+      if (token !~ /^[0-9a-fA-F]{64}|[0-9a-zA-Z_\-]{71}$/) && ENV['APPOPTICS_COLLECTOR'] != "sslcollector:12222"
```

Comment:
it's trivial but you don't need the `\` when the `-` is the last character in brackets.